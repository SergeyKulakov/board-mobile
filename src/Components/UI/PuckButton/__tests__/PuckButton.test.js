import React from 'react'
import { shallow } from 'enzyme'

import 'jest-styled-components'

import Component from '../PuckButton'

describe('PuckButton', () => {
  const defaultProps = {
    onClick: () => null,
    style: {},
    textStyle: {},
    outline: false,
  }

  it('render snapshot', () => {
    const wrapper = shallow(<Component {...defaultProps} />)

    expect(wrapper).toMatchSnapshot()
  })
})
