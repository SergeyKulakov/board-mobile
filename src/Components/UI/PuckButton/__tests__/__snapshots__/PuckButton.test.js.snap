// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PuckButton render snapshot 1`] = `
<Styled(TouchableOpacity)
  onPress={[Function]}
  outline={false}
  style={Object {}}
>
  <Styled(Text)
    outline={false}
    style={Object {}}
  />
</Styled(TouchableOpacity)>
`;
